examples/pxScene2d/src/pxShaderResource.h

```diff
@@ -0,0 +1,185 @@
+/*
+
+pxCore Copyright 2005-2018 John Robinson
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  Created by Fitzpatrick, Hugh on 5/10/19.
+*/
+
+#ifndef PXSHADERRESOURCE_HPP
```

Comment:
Please name to `PXSHADERRESOURCE_H` (from `PXSHADERRESOURCE_HPP`) since this is a .h file (and not a .hpp file).

Reply:
Done